cmd/generic-handlers.go

```diff
@@ -723,6 +723,21 @@ func (l rateLimit) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	l.handler.ServeHTTP(w, r)
 }
 
+type requestIDHeaderHandler struct {
+	handler http.Handler
+}
+
+func addrequestIDHeaders(h http.Handler) http.Handler {
+	return requestIDHeaderHandler{handler: h}
+}
+
+func (s requestIDHeaderHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	header := w.Header()
+	// Set unique request ID for each response.
+	header.Set(responseRequestIDKey, mustGetRequestID(UTCNow()))
```

Comment:
perhaps write it in one line to avoid the confusion of map copy. 

```
w.Header().Set(responseRequestIDKey, mustGetRequestID(UTCNow()))
```

Reply:
done